README.md

```diff
@@ -32,22 +33,231 @@ scheduled_task { 'Run Notepad':
 
 ## Usage
 
-See the [Puppet resource documentation](https://puppet.com/docs/puppet/latest/types/scheduled_task.html) for more information.
+One common use is to kick off a script, either once or on a regular cadence.
+In this first example we schedule a cleanup script to run this one time.
+
+~~~ puppet
+scheduled_task { 'Disk Cleanup': # Unique name for the scheduled task
+  command => 'C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe -File "C:\\Scripts\Clear-DiskSpace.ps1"',
+  enabled => 'true',             # This is the default, but including it is good practice. Flip to 'false' to disable the task.
+  trigger => [{
+    schedule   => 'once',        # Defines the trigger type; required.
+    start_time => '23:20',       # Defines the time the task should run; required.
+    start_date => '2018-01-01'   # Defaults to the current date; not required.
+  }],
+}
+~~~
+
+If we need to have the cleanup script run every night we can use a daily trigger.
+Just changing the trigger schedule from `once` to `daily` will do the trick.
+
+~~~ puppet
+scheduled_task { 'Disk Cleanup Nightly':
+  command => 'C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe -File "C:\\Scripts\Clear-DiskSpace.ps1"',
+  enabled => 'true',
+  trigger => [{
+    schedule         => 'daily',
+    start_time       => '23:20'
+  }],
+}
+~~~
+
+You can also set your scheduled tasks to repeat during an set time block.
+Re-using the idea of a cleanup script, this scheduled task begins at the same time every day and runs once an hour from seven in the morning to seven at night as the SYSTEM account.
+
+~~~ puppet
+scheduled_task { 'Disk Cleanup Daily Repeating':
+  ensure  => 'present',
+  command => 'C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe -File "C:\\Scripts\Clear-DiskSpace.ps1"',
```

Comment:
Is the single slash at end of the file path correct? I don't know if this was a deliberate decision or not, but I won't bother commenting on each instance.

Reply:
Nah, bug on my part. Good catch!